api-server/common/models/User-Identity.js

```diff
@@ -43,7 +43,7 @@ export default function(UserIdent) {
     // get the email from the auth0 (its expected from social providers)
     const email =
       profile && profile.emails && profile.emails[0]
-        ? profile.emails[0].value
+        ? profile.emails[0].value.toLowerCase()
```

Comment:
I should have clarified, this may not be guarenteed to be a string.

Reply:
So, if the value is there, it still may return as `undefined` or similar?